eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -8,9 +8,8 @@ description:
   via Plutus Core.
 bug-reports:         https://github.com/input-output-hk/cardano-ledger/issues
 license:             Apache-2.0
-author:              IOHK Formal Methods Team
-maintainer:          formal.methods@iohk.io
-copyright:           2020 Input Output (Hong Kong) Ltd.
```

Comment:
I wonder if Jared's [comment](https://github.com/input-output-hk/cardano-ledger/issues/2999#issuecomment-1227475577)   
 > be consistent with the packages in the cardano-node repository with respect to the maintainer email and author 

implies that we should get rid of the copyright, or it refers only to these two fields :thinking:

Reply:
Makes sense :+1: